src/Components/UI/NoteLabel/NoteLabel.jsx

```diff
@@ -1,13 +1,18 @@
-import { usePebbleNote } from "../../../Context";
+import { useAlert, usePebbleNote } from "../../../Context";
 import ButtonSimple from "../Button/ButtonSimple";
 import "./NoteLabel.css";
 
 const NoteLabel = () => {
   const { label, setLabel, dispatch } = usePebbleNote();
+  const { alertDispatch } = useAlert();
 
   const submitLabel = (e) => {
     e.preventDefault();
-    dispatch({ type: "addLabelToNote", payload: label });
+    if (label.trim() === "") {
+      alertDispatch({ type: "emptyLael" });
```

Comment:
Please check the type. Spelt wrong.

Reply:
Thank you for pointing it out. Will Fix it.